content/content.go

```diff
@@ -117,11 +155,14 @@ func (s *RulesWithContentStorage) ResetContent() {
 	s.Lock()
 	defer s.Unlock()
 
-	s.rulesWithContent = make(map[ruleIDAndErrorKey]*local_types.RuleWithContent)
 	s.rules = make(map[types.RuleID]*types.RuleContent)
+	s.rulesWithContent = make(map[ruleIDAndErrorKey]*local_types.RuleWithContent)
+	s.recommendationsWithContent = make(map[types.RuleID]*local_types.RuleWithContent)
+	s.internalRuleIDs = make([]types.RuleID, 0)
```

Comment:
actually you can leave it as `nil` as the first `assert` will create a slice for you (nitpick)